code/src/components/Header.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import moment from 'moment'
+import { todos } from "reducers/todos";
+import { useSelector, useDispatch } from "react-redux";
+
+export const Header = () => {
+  const list = useSelector((store) => store.todos.list);
+  const doneList = list.items.filter((item) => item.done);
```

Comment:
if you're just interested in the length of this list, you could store only that in doneList variable.
 `const doneList = list.items.filter((item) => item.done).length`